overlord/snapstate/backend/link.go

```diff
@@ -68,6 +68,10 @@ func (b Backend) LinkSnap(info *snap.Info, model *asserts.Model) error {
 		return err
 	}
 
+	if err := updateFontconfigCaches(); err != nil {
```

Comment:
Should this take the snapInfo and be constrained to snaps installed with classic confinement?

Reply:
That is an interessting question - it would have to be classic and snaps that use the desktop or fontconfig interface. We could limit to those snaps, I'm open for suggestions. Right now its unconditionally as its really fast but maybe that is a mistake?